e2e/scenario_test.go

```diff
@@ -649,6 +649,31 @@ func Test_ubuntu2204(t *testing.T) {
 	})
 }
 
+func Test_ubuntu2204Scriptless(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Description: "Tests that a node using the Ubuntu 2204 Scriptless VHD can be properly bootstrapped",
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2204Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
```

Comment:
Does it do anything? It looks like `*datamodel.NodeBootstrappingConfiguration` isn't used in your changes.

Reply:
I use the existing nbc to populate the new contract in `baseNodeBootstrappingContract` in `templates.go`. However,  I think you're right that these overrides aren't doing anything.